templates/repo/issue/labels/label_list.tmpl

```diff
@@ -33,6 +33,11 @@
 		<li class="item">
 			<div class="label-title">
 				{{RenderLabel $.Context .}}
+				{{if not .ArchivedUnix.IsZero}}
+					<i class="gt-ml-3 gt-cursor-default" data-tooltip-content="{{$.locale.Tr "repo.issues.label_archive_tooltip"}}">
+						{{$.locale.Tr "home.show_archived"}}
+					</i>
```

Comment:
I think we better introduce a `gt-italic` helper class instead of using `<i>` (which btw is/was abused for icons in the past, still some remnants of it in the code).

Reply:
I was searching for this css.